src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -1370,20 +1365,6 @@ intx ObjectMonitor::complete_exit(JavaThread* current) {
   return save;
 }
 
-// reenter() enters a lock and sets recursion count
-// complete_exit/reenter operate as a wait without waiting
-bool ObjectMonitor::reenter(intx recursions, JavaThread* current) {
```

Comment:
AFAICS `complete_exit` is also unused now

Reply:
It is still used here: // Monitor cleanup on JavaThread::exit

Reply:
Doh! Thanks missed ReleaseJavaMonitorsClosure :(